app/models/user.rb

```diff
@@ -633,21 +629,6 @@ def create_conditional_autovomits
     Spam::Handler.handle_user!(user: self)
   end
 
-  # TODO: @citizen428 I don't want to completely remove this method yet, as we
-  # have similar methods in other models. But the previous implementation used
-  # three profile fields that we can't guarantee to exist across all Forems. So
-  # for now this method will just return an empty string.
-  def tag_keywords_for_search
-    ""
-  end
-
-  # TODO: this can be removed once we migrate away from ES
```

Comment:
We migrated aways from ES a long time ago, so this should go now, `User` is big enough without carrying any extra ballast.